dist/marky-markdown.js

```diff
@@ -672,16 +672,16 @@ sanitizer.config = {
     h4: ['id'],
     h5: ['id'],
     h6: ['id'],
-    a: ['href', 'id', 'name', 'target'],
-    img: ['id', 'src', 'width', 'height', 'valign'],
+    a: ['href', 'id', 'name', 'target', 'title'],
+    img: ['id', 'src', 'width', 'height', 'valign', 'title'],
     meta: ['name', 'content'],
     iframe: ['src', 'frameborder', 'allowfullscreen'],
     input: ['checked', 'class', 'disabled', 'type'],
     div: ['id'],
     span: [],
     pre: [],
-    td: ['colspan', 'rowspan'],
-    th: ['colspan', 'rowspan'],
+    td: ['colspan', 'rowspan', 'style'],
+    th: ['colspan', 'rowspan', 'style'],
```

Comment:
why is this in here? wasn't this from https://github.com/npm/marky-markdown/pull/216/files? i'm confused because changes to `td` and `th` aren't in the desc for this. unless i'm missing something...